lib/puppet/type/aem_package.rb

```diff
@@ -27,6 +27,15 @@
       nil
     end
 
+    newvalue(:reinstall) do
```

Comment:
Since it's a state, please rename `reinstall` to `reinstalled`, consistent with `archived` .